index.js

```diff
@@ -0,0 +1,19 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nave-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
- Kindly check this functionality, since while I'm using the navigation menu on the mobile version, after going to a section, when I want to use again the hamburger button it's doesn't work anymore.